tslint.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "typestrict",
+  "rules": {
+    "no-console": true,
+    "no-debugger": true,
+    "mocha-avoid-only": true,
+    "prefer-const": true,
+    "no-var-keyword": true,
+    "interface-name": [true, "never-prefix"],
+    "no-commented-code": true,
```

Comment:
Again, don't find this rule in the docs.

Reply:
It's defined here: https://github.com/SonarSource/SonarTS

Reply:
If these rules are defined "somewhere" around in the ecosystem, how come that they are included in the generic TS code base? Or did you add dependencies for that?

Reply:
These additional rulesets are part of `typestrict` so tslint just finds them 🤷‍♂️ I agree that we could add them to dev deps as well...

Reply:
Generally I think this rule make sense, commented out code (side node: this rule would be self-descriptive if just named "no-commented-out-code", just another reminder on how important names are) tends to very much be left and float around forever and putting additional noise to the code base.

Reply:
Ah no, don't let's bloat up the dev dependencies on this, but we should find ways to make the fact transparent that rules are also coming from there.

Actually I will start on an organizational ReadTheDocs documentation on the ``organization`` repo in the coming days, including things like contribution guidelines and also some tech standards.

I will also include a TypeScript section there and try to extract from here the common set of practices and tools we agreed upon. This guide can the referenced from the various repos.

Reply:
(ah, and maybe to the point: we can add some additional explanation on TypeStrict there as well.)

Reply:
Yeah, I will update the readme of `typestrict`.